src/mono/wasm/runtime/http.ts

```diff
@@ -56,6 +58,7 @@ export function http_wasm_fetch(url: string, header_names: string[], header_valu
     }
 
     return wrap_as_cancelable_promise(async () => {
+        const fetch = imports.fetch || globalThis.fetch || fetch_like;
```

Comment:
`fetch_like` is preferred for assets because it could also handle local files.
But for C# http `fetch_like` is not good enough.
I added `runtimeHelpers.fetch` and it's nice, thanks!